generator/src/main/java/com/scottlogic/deg/generator/validators/ConfigValidator.java

```diff
@@ -4,6 +4,7 @@
 import com.scottlogic.deg.generator.generation.GenerationConfig;
 import com.scottlogic.deg.generator.generation.GenerationConfigSource;
 import com.scottlogic.deg.generator.outputs.targets.OutputTarget;
+import com.scottlogic.deg.schemas.common.ValidationResult;
```

Comment:
I guess this import isn't needed?

Reply:
as ValidationResult was needed for the ProfileSchemaValidator, I copied it into the schemas jar.
It made sense to use that one as we already include the schemas.jar file